### ISSUE TITLE
[Enhancement]:

### ISSUE THREAD

**Issue by MSC-lab** [woothemes/woocommerce#53478]
### Describe the solution you'd like

In June 2025,

a new law on website accessibility will come into force in the European Union. An initial review of your plugin has identified many problems in the implementation of the required functions. Are you aware of the legal adjustment and are you planning to make adjustments to the plugin by June 2025?

Thank you for your feedback!

**Comment by rodelgc** [comment]
Thank you for raising this @MSC-lab. We'll look into this. 

~~For the meantime, could you provide a more meaningful issue title? Currently it's just titled `[Enhancement]:`. Thanks!~~

**Comment by rodelgc** [comment]
Hi @MSC-lab to answer your question, Yes, we are already aware of the European Accessibility Act, and there has already been a number of internal discussions to comply with it. I'll close this issue for now. If you would like to know more, feel free to ask on our support channels instead, in order to keep GitHub as a place for treating bugs and new enhancement requests.

- If you are a paying customer of WooCommerce, contact WooCommerce support by [opening a ticket or starting a live chat](https://woocommerce.com/contact-us/);
- Make a post on [WooCommerce community forum](https://wordpress.org/support/plugin/woocommerce/)
- Join the [WooCommerce Community Slack channel](https://woocommerce.com/community-slack/).

**Comment by rodelgc** [comment]
Hi @MSC-lab maybe this article would also help in answering where we're at in addressing the accessibility issues - [Elevating Ecommerce Accessibility: An Interview with Equalize Digital and WooCommerce’s Progress](https://developer.woocommerce.com/2024/08/09/elevating-ecommerce-accessibility-an-interview-with-equalize-digital-and-woocommerces-progress/). Hope this helps!
